frontend/src/components/AvatarComponent.js

```diff
@@ -1,5 +1,9 @@
 import React, { useState, useEffect } from "react";
 import axios from "axios";
+import "bootstrap/dist/css/bootstrap.min.css";
+import Form from "react-bootstrap/Form";
+import InputGroup from "react-bootstrap/InputGroup";
+import "./AvatarComponent.css";
 
 const AvatarComponent = (props) => {
   const [seed, setSeed] = useState(props.seed);
```

Comment:
이것도 좋긴 한데, 
```suggestion
const AvatarComponent = ({ seed: seedProp }) => {
  const [seed, setSeed] = useState(seedProp);
```

이렇게 하면 변수 이름 충돌이 없으면서도, 어떤게 prop으로 들어가야하는지 명확하겠지? 일반적인 사람들은 `props.seed`를 보기 전까지는 props에 어떤 것이 들어가는지를 모를테니까!

물론 이것도 typescript로 바뀌면 사라질 문제긴해~